### PR TITLE
Version bump 1.5.0 + crates update

### PR DESCRIPTION
**Pull request by gRoussac**
Version bump 1.5.0 + crates update

### CODE REVIEW

**Review by ACStone-MTS — COMMENTED** [review]
Docs portion LGTM, leaving approval for tech reviews.

**Review by bpr — APPROVED** [review]
LGTM

File: client-js/CHANGELOG.md [hunk 1]
```diff
@@ -7,6 +7,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [1.5.0] - 2023-10-05
+
```
Comment (hoffmannjan) [inline client-js/CHANGELOG.md]:
```suggestion
- `ces-js-parser` is not as a fixed version
```

**Review by hoffmannjan — APPROVED** [review]
One addition in changelog and otherwise LGTM

**Pull request merged**